diff --git a/node_modules/react-native-select-dropdown/src/styles.js b/node_modules/react-native-select-dropdown/src/styles.js
index d4d0d61..727dc29 100644
--- a/node_modules/react-native-select-dropdown/src/styles.js
+++ b/node_modules/react-native-select-dropdown/src/styles.js
@@ -7,7 +7,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     backgroundColor: '#EFEFEF',
     width: width / 2,
-    height: 50,
+    // height: 50,
     paddingHorizontal: 8,
     overflow: 'hidden',
   },
